R/to-html.r

```diff
@@ -195,10 +195,11 @@ to_html.item <- function(x, ...) {
 #' @export
 to_html.eqn <- function(x, pkg, ...) {
   stopifnot(length(x) <= 2)
```

Comment:
I think so - it will catch bad use of `\eqn{}` — I think it's still reasonable to check